map_view/display-device-location-with-autopan-modes/README.metadata.json

```diff
@@ -12,26 +12,24 @@
         "map",
         "mobile",
         "navigation",
-        "ArcGISMap",
         "LocationDisplay",
         "LocationDisplay.AutoPanMode",
         "MapView",
         "SimulatedLocationDataSource",
         "SimulationParameters"
     ],
     "redirect_from": [
-        "/java/latest/sample-code/display-a-map.htm"
+        ""
     ],
```

Comment:
I wonder if the script for generating this is broken again - can you check please? @JonLavi might be able to help. 

We must *not* have an empty string within an array: it will break the website build, so this small change is important. it needs to be: 

```
"redirect_from": "",
```